src/login/login.component.html

```diff
@@ -1,4 +1,4 @@
-<md-content class="md-hue-2 login-bg"
+<md-content class="md-hue-2 login-social-bg"
```

Comment:
I think this belongs to the whole login component, not just the social auth part (email based auth is also included). So why the new name?

Reply:
Please see the commit comment, the new name is just to avoid reusing the same class as metabolica-ui login component. On load, I've noticed sometimes the metabolica-ui background is displayed for a second or two before our background takes precedence so this is a quick fix to avoid that. The proper solution would be to not load the metabolica-ui login component at all.